IBPSA/Utilities/Math/MovingAverage.mo

```diff
@@ -0,0 +1,187 @@
+  // an if-then construct
+  when time >= tStart+delta then
+    mode=true;
+  end when;
+  if mode then
+    y=(mu-muDel)/delta;
+  else
+    y=(mu-muDel)/(time-tStart+1E-3);
+  end if;
+  annotation (
+    defaultComponentName="movAve",
+    Icon(
+      graphics={
+        Rectangle(
+          extent={{-100,-100},{100,100}},
+          lineColor={0,0,127},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Polygon(
+          points={{-78,90},{-86,68},{-70,68},{-78,90}},
+          lineColor={192,192,192},
+          fillColor={192,192,192},
+          fillPattern=FillPattern.Solid),
+        Line(
+          points={{-78,68},{-78,-80}},
+          color={192,192,192}),
+        Line(
+          points={{-88,0},{70,0}},
+          color={192,192,192}),
+        Polygon(
+          points={{92,0},{70,8},{70,-8},{92,0}},
+          lineColor={192,192,192},
+          fillColor={192,192,192},
+          fillPattern=FillPattern.Solid),
+        Line(
+          points={{-78,-31},{-64,-31},{-64,-15},{-56,-15},{-56,-63},{-48,-63},
+          {-48,-41},{-40,-41},{-40,43},{-32,43},{-32,11},{-32,11},{-32,-49},
+          {-22,-49},{-22,-31},{-12,-31},{-12,-59},{-2,-59},{-2,23},{4,23},
+          {4,37},{10,37},{10,-19},{20,-19},{20,-7},{26,-7},{26,-37},
+          {36,-37},{36,35},{46,35},{46,1},{54,1},{54,-65},{64,-65}},
+          color={215,215,215}),
+        Line(
+          points={{-78,-24},{68,-24}}),
+        Text(
+          extent={{-140,152},{160,112}},
+          textString="%name",
+          textColor={0,0,255}),
+        Text(
+          extent={{-42,-63},{41,-106}},
+          textColor={192,192,192},
+          textString="%delta s"),
+        Text(
+          extent={{226,60},{106,10}},
+          textColor={0,0,0},
+          textString=DynamicSelect("",String(y,
+            leftJustified=false,
+            significantDigits=3)))}),
+    Documentation(
+      info="<html>
+<p>
+This block outputs the mean value of its input signal as
+</p>
+<pre>
+      1  t
+y =   -  &int;   u(s) ds
+      &delta;  t-&delta;
+</pre>
+<p>
+where <i>&delta;</i> is a parameter that determines the time window over
+which the input is averaged.
+For
+<i> t &lt; &delta;</i> seconds, it outputs
+</P>
+<pre>
+           1      t
+y =   --------    &int;   u(s) ds
+      t-t<sub>0</sub>+10<sup>-10</sup>   t<sub>0</sub>
+</pre>
+<p>
+where <i>t<sub>0</sub></i> is the initial time.
+</p>
+<p>
+This block can for example be used to output the moving
+average of a noisy measurement signal.
+</p>
+<p>
+See
+<a href=\"modelica://Buildings.Controls.OBC.CDL.Continuous.Validation.MovingAverage\">
+Buildings.Controls.OBC.CDL.Continuous.Validation.MovingAverage</a>
+and
+<a href=\"modelica://Buildings.Controls.OBC.CDL.Continuous.Validation.MovingAverage_nonZeroStart\">
+Buildings.Controls.OBC.CDL.Continuous.Validation.MovingAverage_nonZeroStart</a>
+for example.
+</p>
+</html>",
+revisions="<html>
```

Comment:
Updat revision history

Reply:
@amaccarini you didn't update the revision history because the component is copied from the Buildings library?

Reply:
I think I forgot to reply to this message, but yes, I copied this component from the Buildings library, so not sure what's the best approach about revision history.

Reply:
Let's leave it like this then. @mwetter could you confirm?

Reply:
@mwetter, not sure I understand. Are you suggesting we delete the block from `IBPSA.Utilities.Math` and move its source code to the protected section within the examples?